modules/nf-core/hmmer/hmmrank/main.nf

```diff
@@ -0,0 +1,70 @@
+process HMMER_HMMRANK {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-b2ec1fea5791d428eebb8c8ea7409c350d31dada:a447f6b7a6afde38352b24c30ae9cd6e39df95c4-1' :
+        'biocontainers/mulled-v2-b2ec1fea5791d428eebb8c8ea7409c350d31dada:a447f6b7a6afde38352b24c30ae9cd6e39df95c4-1' }"
+
+    input:
+    tuple val(meta), path(tblouts)      // HMMER_HMMSEARCH.out.target_summary
+
+    output:
+    tuple val(meta), path("*.hmmrank.tsv.gz"), emit: hmmrank
+    path "versions.yml"                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    #!/usr/bin/env Rscript
+    library(readr)
+    library(dplyr)
+    library(tidyr)
+    library(stringr)
+
+    # Read all the tblout files
+
+    read_fwf(c('${tblouts.join("','")}'), fwf_cols(content = c(1, NA)), col_types = cols(content = col_character()), comment='#', id = 'fname') %>%
+        filter(! grepl('^ *#', content)) %>%
```

Comment:
if you are using `stringr` anyway, why not use `str_detect`?

Reply:
Good point. Old code.

Reply:
Good point. Old code. (I thought I already replied to this one, but can't see that...) Anyway, updated now after I got the test to actually do something.